### PR TITLE
feat(devx): More & better startup

### PR DESCRIPTION
**Pull request by kristofer-lp8f**
## What
Improve DevX for `api`, `web`, `application-system-api`, and `application-system-form`.

## Checklist:

- [ ] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] Formatting passes locally with my changes
- [ ] I have rebased against main before asking for a review

### CODE REVIEW

File: apps/api/README.md [hunk 1]
```diff
@@ -1,5 +1,21 @@
 # API
 
+## Quickstart
+
+Ensure docker is running, then run the following to for the first time:
```
Comment (RunarVestmann) [inline apps/api/README.md:5]: Is there a typo here?

File: apps/application-system/README.md [hunk 2]
```diff
@@ -1,26 +1,28 @@
 # Application System
 
-## About
-
-This project forms the base for all business applications belonging to island.is.
-
 ## Quickstart
 
-Ensure docker is running, then run the following to set up:
+Ensure docker is running, then run the following to for the first time:
```
Comment (RunarVestmann) [inline apps/application-system/README.md:5]: Same here, that "to" feels off place

File: apps/web/README.md [hunk 3]
```diff
@@ -1,5 +1,21 @@
 # Web
 
+## Quickstart
+
+Ensure docker is running, then run the following to for the first time:
```
Comment (RunarVestmann) [inline apps/web/README.md:5]: .

**Review by RunarVestmann — COMMENTED** [review]
A few minor comments regarding text.
How does this affect the web project exactly?
What does yarn dev api mean?

File: apps/api/README.md (hunk 1 above)
Comment (kristofer-lp8f) [reply apps/api/README.md:5]: Yes, a typo.

File: apps/application-system/README.md [hunk 4]
```diff
@@ -1,26 +1,28 @@
...
 
-This command runs all _initial setup_ in the sections [API](#API) and [Form](#Form) below.
-
-Then whenever you want to work on the application-system run this command:
+To start the app:
 
 ```bash
-yarn nx run application-system-api:dev
+yarn dev application-system-api
+or
```
Comment (Thorrik58) [inline apps/application-system/README.md:17]: Should probably be and not or. Both need to be running.

**Comment by kristofer-lp8f** [comment]
> What does yarn dev api mean?

TL;DR: `dev`="start now plz", `dev/init`="take care of initial setup for me"

This PR is improving Developer Experience by simplifying how we start a project. `yarn dev api` is just an "alias" for running all the required commands/steps in order to run `api`. It's redundant for `api`, but I value the consistency.

With this, going into a new project (e.g. when reviewing, testing, starting work on) getting started should be much easier. If you need the output of a specific command, you can run it first, then `yarn dev <project>` in another window. The "duplicate" command will fail due to port collision, so you're never running multiple instances of the app you need a separate console for.

**Review by RunarVestmann — APPROVED** [review]

**Review by eirikurn — APPROVED** [review]
❤️

**Review by rafnarnason — APPROVED** [review]
lgtm

**Review by obmagnusson — APPROVED** [review]
LGTM
